bundles/org.openhab.binding.mcp23017/README.md

```diff
@@ -4,6 +4,9 @@ This binding allows you to have native access for MCP23017 I/O expander on I2C b
 It was tested with Raspberry Pi 2 and Raspberry Pi 3, but probably should work with other devices supported by [Pi4J](https://pi4j.com/) library.
 
 On Raspberry Pi the user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
+As the MCP23017 has 3 address pins, you are restricted to 8 devices on a I2C bus.
+To use more devices you have to open further I2C busses. Therefore you can use overlays to enable bit banging I2C busses on the Raspberry Pi connector, up to I2C6.
```

Comment:
Please use a new line for each sentence. I also don't understand what excatly is needed to enable the overlays. Is this just a configuration option the user has to set?

Reply:
To be more precise: it is a Linux kernel device tree overlay which can be appended in the /boot/config.txt file of RFS. It is also a configuration of alternative pin functions. I´ve added therefore  the link to available overlays

Reply:
In my case I ´ve added  
dtoverlay=i2c-gpio,i2c_gpio_sda=20,i2c_gpio_scl=21,i2c_gpio_delay_us=1
to config.txt s. [examples](https://www.instructables.com/id/Raspberry-PI-Multiple-I2c-Devices/)

Reply:
Thanks. That makes it more clear